src/Controller/FileVariantController.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace GoApptiv\FileManagement\Controller;
+
+use Exception;
+use Illuminate\Http\Request;
+use GoApptiv\FileManagement\Services\FileManagement\FileManagementService;
+use Illuminate\Routing\Controller;
+use Illuminate\Support\Facades\Log;
+use GoApptiv\FileManagement\Constants;
+use GoApptiv\FileManagement\Requests\FileVariantRequest;
+use GoApptiv\FileManagement\Events\ConfirmFileVariantEvent;
+use GoApptiv\FileManagement\Models\FileManagement\Dto\FileManagementVariantLogData;
+
+class FileVariantController extends Controller
+{
+    /** @var FileManagementService */
+    private $fileManagementService;
+
+    /**
+     * Constructor.
+     *
+     * @param FileManagementService $fileManagementService
+     */
+    public function __construct(FileManagementService $fileManagementService)
+    {
+        $this->fileManagementService = $fileManagementService;
+    }
+
+    /**
+     * Update Variant Id and status by UUID
+     *
+     * @param Request $request
+     *
+     * @return Object
+     *
+     */
+    public function updateVariantDetails(FileVariantRequest $request)
+    {
+        try {
+            $requestData = $request->getData();
+            $dataString = $requestData->getDataString();
+
+            $decodedDataString = base64_decode($dataString, true);
+            $json = json_decode($decodedDataString, true);
+            Log::info(json_last_error());
```

Comment:
json_last_error? What would be output of this? and is it required in the log message

Reply:
Add the context